static/js/api.js

```diff
@@ -34,6 +36,11 @@ var _handleOutboundCalls = function _handleOutboundCalls(e, ace) {
       authorsColors.setUsersColors(e.data.usersColors);
       break;
     }
+
+    case ETHERPAD_RECONNECTED: {
+      _triggerEvent({ type: GET_USERS_COLORS });
```

Comment:
a gente deixa essa separação entre mensagens que entram e saem bem definidas. Veja por exemplo, https://github.com/storytouch/ep_comments/blob/master-tk/static/js/api.js
A API aqui poderia conhecer menos da lógica. Ela poderia somente receber a ação e deixar que o módulo que trata das cores fizesse o que for necessário. Isso seria legal por que imagino que esse problema vai acabar acontecendo em outros lugares também - agora é um problema das cores mas mais para frente esse event `ETHERPAD_RECONNECTED` pode deixar de ser escutado aqui. Isso facilita um teste do plugin também.